packages/terra-list/src/ListItem.jsx

```diff
@@ -54,13 +54,18 @@ const propTypes = {
    * @private Callback function not intended for use with this API, but if set pass it through to the element regardless.
    */
   onMouseDown: PropTypes.func,
+  /**
+   * Set to true for non-selectable items in a list having role as listbox.
+   */
+  isRoleListbox: PropTypes.bool,
```

Comment:
This is confusing to me. 

- Why do we need this prop at all?
- Don't we recommend using 'listbox' as the role for all lists for accessibility? If not, why?
- The description 'Set to true for non-selectable items in a list having role as listbox.' doesn't sound right to me.

Reply:
Listbox is set as role for list only if it has items that are selectable as described [here](https://github.com/cerner/terra-core/blob/01cba497294d6b055d4482275a1689b85cef52ca/packages/terra-list/src/List.jsx#L28)